src/django_browser_reload/views.py

```diff
@@ -154,7 +154,10 @@ def event_stream() -> Generator[bytes, None, None]:
                 should_reload_event.clear()
                 yield message("reload")
 
-    return StreamingHttpResponse(
+    response = StreamingHttpResponse(
         event_stream(),
         content_type="text/event-stream",
     )
+    # Set a Content-Encoding to bypass GzipMiddleware
+    response["Content-Encoding"] = ""
```

Comment:
[Another way](https://github.com/django/django/blob/c3d7a71f836f7cfe8fa90dd9ae95b37b660d5aae/django/middleware/gzip.py#L23-L31) would be to set `request.META["HTTP_ACCEPT_ENCODING"] = ""`. But that may break if GzipMiddleware stars reading from `request.headers` because [it is a `@cached_property`](https://github.com/django/django/blob/c3d7a71f836f7cfe8fa90dd9ae95b37b660d5aae/django/http/request.py#L102-L104).

Let me know what you'd prefer.

Reply:
That is more hacky. Let's stick to an empty content-encoding.

We can use lowercase header names as per http/2.

```suggestion
    # Set a content-encoding to bypass GzipMiddleware etc.
    response["content-encoding"] = ""
```